ipams/builtin/builtin_unix.go

```diff
@@ -11,6 +11,11 @@ import (
 	"github.com/docker/libnetwork/ipamutils"
 )
 
+var (
+	// DefaultAddressPool Store DefaultAddressPool locally to use it during IPAM utils init
+	DefaultAddressPool []*ipamutils.PredefinedPools
```

Comment:
this can be private, you have the setter and should be used only internally?

Reply:
made it private